planet/src/main/java/com/google/planet/servlets/ItineraryServlet.java

```diff
@@ -45,6 +45,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         PreparedQuery results = datastore.prepare(query);
 
+        // Create an event called "hotel" with duration of 0
+        String hotelAddress = request.getParameter("hotel-address");
+        Event hotel = new Event(123, 
+                                "Hotel", 
+                                hotelAddress, 
+                                0, 
+                                TimeRange.WHOLE_DAY, 
+                                "listName-none", 
+                                "userId-none");
```

Comment:
rather than putting garbage data into this API, these parameters should be optional.

Suggest making a separate constructor for Event that omits ID, list name, and user id.

Reply:
Ah Okay!